### PR TITLE
Update the mono-repo to one package

### PR DESCRIPTION
**Pull request by abdonrd**
We want to try to publish the components as a single package in npm.

### CODE REVIEW

**Comment by abdonrd** [comment]
Right now the Storybook is not working:

Screenshot 2021-11-08 at 18 44 31

**Comment by abdonrd** [comment]
Seems that the problem is the resolution of the dependencies. We need to wait to the next `carbon-web-components` release, which stops having dependencies as `peerDependencies`.

File: package.json [hunk 1]
```diff
@@ -23,6 +24,21 @@
...
+    "components/**/*.{d.ts,js,js.map}",
+    "!components/**/*.stories.{d.ts,js,js.map}",
+    "!components/**/*.test.{d.ts,js,js.map}",
+    "!components/scss.d.ts"
+  ],
+  "dependencies": {
+    "@carbon/colors": "^10.34.0",
+    "@carbon/layout": "^10.34.0",
+    "@carbon/type": "^10.38.0",
+    "carbon-components": "~10.47.1",
```
Comment (abdonrd) [inline package.json:37]: Also remove `carbon-components` when update `carbon-web-components`.

File: package.json [hunk 2]
```diff
@@ -23,6 +24,22 @@
...
+    "components/**/*.{d.ts,js,js.map}",
+    "!components/**/*.stories.{d.ts,js,js.map}",
+    "!components/**/*.test.{d.ts,js,js.map}",
+    "!components/scss.d.ts"
+  ],
+  "dependencies": {
+    "@carbon/colors": "^10.34.0",
+    "@carbon/layout": "^10.34.0",
+    "@carbon/type": "^10.38.0",
+    "carbon-web-components": "file:carbon-web-components-v1.19.0-beta.tgz",
```
Comment (abdonrd) [inline package.json:37]: Wait for the release before merging the PR.

File: package.json (hunk 1 above)
Comment (abdonrd) [reply package.json]: > Right now the Storybook has stopped working:
> 
> Screenshot 2021-11-08 at 18 44 31

It's working now! 🎉

**Comment by abdonrd** [comment]
> Right now the Storybook has stopped working:
> 
> Screenshot 2021-11-08 at 18 44 31

It's working now! 🎉

**Review by Tansito — APPROVED** [review]

**Pull request merged**